.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.swift]
```

Comment:
Should we add `root = true` at the top, as the documentation suggests?

Reply:
Yes, most definitely! That was an oversight on my part.